event_webhook_types_test.go

```diff
@@ -217,8 +216,8 @@ func TestMergeEventUnmarshal(t *testing.T) {
 		t.Errorf("BlockingDiscussionsResolved isn't true")
 	}
 
-	if event.Assignees[0].ID != expectedID {
-		t.Errorf("Assignees[0].ID is %v, want %v", event.Assignees[0].ID, expectedID)
+	if event.Assignees[0].ID != 6 {
```

Comment:
Please use the constant (and make sure that the test input uses the same values)...